config/packages/doctrine_migrations.yaml

```diff
@@ -0,0 +1,6 @@
+doctrine_migrations:
```

Comment:
Are you sure this file is required ? I launch the migrations yesterday without this configuration file and it's worked

Reply:
isn't needed for a migration (d:m:m) but needed for create the migration file (make:migration)

Reply:
but u can remove the debug line x)